### PR TITLE
Update dependencies on gems with known security issues

### PR DESCRIPTION
**Pull request by unleashed**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,11 +1,11 @@
 source 'https://rubygems.org'
 
 ruby '2.3.1'
-gem 'rack', '1.6.5'
-gem 'sinatra'
+gem 'rack', '~> 1'
```
Comment (mikz) [inline Gemfile]: This requirement is good as having nothing there really :)

File: Gemfile [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+gem 'sinatra', '~> 1'
 gem 'puma'
-gem 'nokogiri', '1.6.8.1'
-gem 'rack-cors', :require => 'rack/cors'
+gem 'nokogiri', '~> 1.8.1'
```
Comment (mikz) [inline Gemfile]: I think we could just drop the requirement and rely on `bundle update`.

File: Gemfile (hunk 1 above)
Comment (unleashed) [reply Gemfile]: sure, but I won't be updating to rack 2 - anything can break!

Comment (unleashed) [reply Gemfile]: Ah, I now see your other PR - would be good to merge if it works with rack-cors.

Comment (mikz) [reply Gemfile]: `~> 1` should map to `~> X` no? It would have to be `< 2`.

Comment (unleashed) [reply Gemfile]: yep

Comment (unleashed) [reply Gemfile]: sorry, understood incorrectly: `~> 1` will map to any `1.X.Y` version, below 2.

**Comment by mikz** [comment]
Updated by https://github.com/3scale/echo-api/pulls?q=is%3Apr+is%3Aclosed+label%3Adepfu

**Pull request closed without merging**